pkgs/development/python-modules/image-match/default.nix

```diff
@@ -0,0 +1,32 @@
+{ stdenv, buildPythonPackage, fetchFromGitHub, pytestrunner, scikitimage, six }:
+
+buildPythonPackage {
+  pname = "image-match";
+  version = "1.1.2";
+
+  src = fetchFromGitHub {
+    owner = "ascribe";
+    repo = "image-match";
+    rev = "1c5f3170555540bdf43ff8b8189c4e8c13a8b950";
+    sha256 = "0vlmpidmhkpgdzw2k03x5layhijcrjpmyfd93yv2ls77ihz00ix5";
+  };
+
+  buildInputs = [ pytestrunner ];
+
+  propagatedBuildInputs = [
+    scikitimage
+    six
+  ];
+
+  # ignore elasticsearch requirement due to version incompatibility
+  installFlags = [ "--no-deps" ];
```

Comment:
it's better to patch `setup.py` / `requirements.txt` / whatever file to remove the specific dependency.